zarf.yaml

```diff
@@ -56,7 +56,7 @@ components:
       - source: assets/misc/empty-file
         target: /etc/rancher/k3s/k3s.yaml
         symlinks:
-          - /root/.kube/config
+          - /root/.kube/configkube
```

Comment:
Is this an intentional change? I don't see anything else in the PR that's changed to look at/for a `configkube` file. With this change, initializing zarf with the k3s component serving as the distro only works if you manually create a symlink `ln -s ~/.kube/configkube ~/.kube/config` before running 'zarf init'.

Reply:
Nope, no idea how the at changed, feel free to fix, thanks